pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -299,10 +341,11 @@ where
 	) -> Result<(BalanceOf<T>, InitialPayment<T>), TransactionValidityError> {
 		match call.is_sub_type() {
 			Some(Call::pay_with_capacity { call }) => self.withdraw_capacity_fee(who, call, len),
+			Some(Call::pay_with_capacity_batch_all { calls }) =>
```

Comment:
I think we then unbox the value using the dereference operator * on call (*call). We also would need to clone it.
```
Some(Call::pay_with_capacity { call }) => self.withdraw_capacity(who, &[*call.clone()], len);
or
Some(Call::pay_with_capacity { call }) => self.withdraw_capacity(who, &vec![*call.clone()], len);
```